test/resources/case42_resource_selector/case42_results_policy.yaml

```diff
@@ -0,0 +1,14 @@
+apiVersion: policy.open-cluster-management.io/v1
+kind: ConfigurationPolicy
+metadata:
+  name: case42-selector-results-e2e
+spec:
+  namespaceSelector:
+    include:
+      - case42-e2e-1
```

Comment:
Could you please include an additional included namespace? I want to see the compliance message in the test of what that looks like.

Reply:
Never mind, I see an example with an empty namespace selector which I forgot selects everything.